src/test/regress/expected/prepared_statements_1.out

```diff
@@ -24,3 +24,25 @@ BEGIN;
 (1 row)
 
 ROLLBACK;
+PREPARE xact_repartitioned_prepared AS
+	SELECT count(*) FROM repartition_prepared_test t1 JOIN repartition_prepared_test t2 USING (b);
+BEGIN;
```

Comment:
Hmm, how are the outputs of the same SELECT query differ so much? There seems no concurrent modification to this table

Reply:
The table contains a lot of duplicate rows, so 1 additional row containing `2` matches a lot of other rows.
```
INSERT INTO repartition_prepared_test SELECT i%2, i%3 FROM generate_series(0,24)i;
```

Reply:
Ah, that's cool, nice test, I missed such a possibility :)  I was concerned we somehow skip some local placements or such.

 Actually, we already verify against PG within the arbitrary configs, but yeah anyway.